Button/Button/MainActivity.cs

```diff
@@ -0,0 +1,22 @@
+namespace Mono.Samples.Button;
+
+[Activity (Label = "Button Demo", MainLauncher = true)]
+public class ButtonActivity : Activity
+{
+    int count = 0;
+
+    protected override void OnCreate (Bundle? bundle)
+    {
+        base.OnCreate (bundle);
+
+        // Create your application here
+        Android.Widget.Button button = new (this);
+
+        button.Text = string.Format ("{0} clicks!!", count);
```

Comment:
This sample was probably written before we had the string interpolation C# feature, can you update the above to:
```suggestion
        button.Text = $"{count} clicks!!";
```
Then one other example below. Thanks!

Reply:
Hi Jonathan, I've just updated the code!